LondonBritishLibrary/orient/BLorient8814.xml

```diff
@@ -439,29 +439,24 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <roleName type="title">Baroness</roleName> Zouche</persName>. <date when="1917">13 Oct. 1917</date>.</q>
                                 </item>
                                 <item xml:id="e2">
-                                    <locus target="#65rb"/>
-                                    <desc type="OwnershipNote">The first owner was <persName role="owner" ref="PRS14385WaldaMadhen"/>. 
-                                        His name is inscribed on several folios.</desc>
-                                </item>
-                                <item xml:id="e3">
                                     <locus target="#39rb"/>
                                     <desc type="OwnershipNote">The second owner was <persName role="owner" ref="PRS14386WalattaYohannes"/>.
                                         Her name is inscribed on several folios.</desc>
                                 </item>
-                                <item xml:id="e4">
+                                <item xml:id="e3">
```

Comment:
I mean: owner names mentioned in the text are not "OwnershipNotes". e3 to e6 are not necessary, the info under history is enough.